CodeNow/runnable-angular
# Moved container files to it's own controller

Extracted container files related stuff from `editServerModalDirective`

To test:
Make sure you can do everything with files and additional repos in the container files view.
